model.py

```diff
@@ -0,0 +1,820 @@
+import os
+import re
+import sys
+import json
+import gguf
```

Comment:
This module is missing the part that lets you use the scripts without installing gguf:
```python
if 'NO_LOCAL_GGUF' not in os.environ:
    sys.path.insert(1, str(Path(__file__).parent / 'gguf-py' / 'gguf'))
import gguf
```